opencl-aot/source/main.cpp

```diff
@@ -303,15 +338,100 @@ int main(int Argc, char *Argv[]) {
                                         cl::desc("Set OpenCL build options"),
                                         cl::value_desc("build options"));
 
+  // Adding custom --version implementation.
+  cl::SetVersionPrinter(
+      [](llvm::raw_ostream &OS) { OS << "Intel(R) SDK for OpenCL(TM)\n"; });
```

Comment:
I'm not sure that this is a proper version string for an open-source tool: this project is not related to `Intel(R) SDK for OpenCL(TM)` and such customization should be done downstream if needed

Reply:
Removed this customization.